website/content/docs/internals/telemetry/enable-telemetry.mdx

```diff
@@ -0,0 +1,109 @@
+---
+layout: docs
+page_title: Enable Vault telemetry
+description: >-
+  Step-by-step guide to enabling telemetry gathering with Vault
+---
+
+# Enable Vault telemetry gathering
+
+Collect telemetry data from your Vault installation.
+
+## Before you start
+
+- **You must have Vault 1.14 or later installed and running**.
+- **You must have access to your [Vault configuration](/vault/docs/configuration) file**.
+
+
+## Step 1: Choose an aggregation agent
+
+@include 'telemetry/supported-aggregation-agents.mdx'
+
+## Step 2: Enable at least one audit device
+
+To include audit-related metrics, you must enable auditing on at least one device
+with the `vault audit enable` command. For example, to enable auditing for the
+`file` device and save the logs to `/var/log/vault_audit.log`:
+
+```shell-session
+$ vault audit enable file file_path=/var/log/vault_audit.log
+```
+
+By default, Enterprise installations replicate audit devices to the secondary
+performance nodes in a cluster. To limit performance replication for an audit
+device, use the `local` flag to mark the device as local to the current node:
+
+```shell-session
+$ vault audit enable file -local file_path=/var/log/vault_audit.log
+```
+
+
+## Step 3: Configure telemetry collection
+
+To configure telemetry collection, update the telemetry stanza in your Vault
+configuration with your collection preferences and aggregation agent details.
+
+For example, the following `telemetry` stanza configures Vault with the standard
+telemetry defaults and connects it to a Statsite agent running on the default
+port within a company intranet at `mycompany.statsite`:
+
+```hcl
+telemetry {
+  usage_gauge_period = "10m"
+  maximum_gauge_cardinality = 500
+  disable_hostname = false
+  enable_hostname_label = false
+  lease_metrics_epsilon = "1h"
+  num_lease_metrics_buckets = 168
+  add_lease_metrics_namespace_labels = false
+  filter_default = true
+  
+  statsite_address = "mycompany.statsite:8125"
+}
+```
+
+<Tip heading="Use a prefix filter to reduce the volume of metrics you receive">
+
+  Many metrics solutions charge by the metric. You can use the `prefix_filter`
+  parameter to include and exclude specific values based on metric name to avoid
+  paying for irrelevant information.
+  
+  For example, to limit your telemetry to the core token metrics plus the number
+  of leases set to expire:
+
+  ```hcl
+  telemetry {
+    prefix_filter = ["+vault.token", "-vault.expire", "+vault.expire.num_leases"]
```

Comment:
I haven't used this feature myself, but my read of it is that your description of what would happen is assuming filter_default=false, which is not the default.

Reply:
D'oh! Yeah, I forgot that part. I added the filter_default parameter to the tip to clarify